package.json

```diff
@@ -15,16 +15,19 @@
     "mkdirp": "^0.5.1"
   },
   "devDependencies": {
-    "babel": "^5.8.21",
-    "babel-eslint": "^4.1.2",
-    "eslint": "^1.4.3"
+    "babel-cli": "^6.1.18",
```

Comment:
Is this supposed to be `babel-core`? I thought the CLI was just for cmd line use now.